modules/RTC/BridgeChannel.js

```diff
@@ -276,8 +276,10 @@ export default class BridgeChannel {
             emitter.emit(RTCEvents.DATA_CHANNEL_OPEN);
         };
 
-        channel.onerror = error => {
-            logger.error('Channel error:', error);
+        channel.onerror = event => {
+            if (this._mode !== 'websocket') {
```

Comment:
Maybe add a comment why. I guess what you have in the description of the PR will be enough.